2020/10/07 14:50:17 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.50.36, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.4:9000", host: "192.168.50.122"
2020/10/07 14:50:33 [error] 29#29: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.50.36, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.4:9000", host: "192.168.50.122"
2020/10/07 14:50:35 [error] 29#29: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.50.36, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.19.0.4:9000", host: "192.168.50.122", referrer: "http://192.168.50.122/"
2020/10/07 14:50:41 [error] 29#29: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.50.36, server: localhost, request: "GET /index.php HTTP/1.1", upstream: "fastcgi://172.19.0.4:9000", host: "192.168.50.122"
2020/10/07 15:22:30 [error] 31#31: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.50.36, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.4:9000", host: "192.168.50.122"
2020/10/07 15:23:03 [error] 31#31: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.50.36, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.4:9000", host: "192.168.50.122"
2020/10/07 15:23:03 [error] 31#31: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.50.36, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.19.0.4:9000", host: "192.168.50.122", referrer: "http://192.168.50.122/"
2020/10/07 15:24:14 [error] 31#31: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.50.36, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.4:9000", host: "192.168.50.122"
2020/10/07 15:31:43 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.50.36, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.4:9000", host: "192.168.50.122"
2020/10/07 15:31:45 [error] 28#28: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.50.36, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.4:9000", host: "192.168.50.122"
2020/10/07 15:39:38 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.50.36, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.4:9000", host: "192.168.50.122"
2020/10/07 15:39:39 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.50.36, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.19.0.4:9000", host: "192.168.50.122", referrer: "http://192.168.50.122/"
2020/10/24 14:33:25 [error] 30#30: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.50.36, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "192.168.50.122"
2020/10/24 20:15:23 [error] 30#30: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.50.36, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "192.168.50.122"
2020/10/24 20:15:23 [error] 30#30: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.50.36, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "192.168.50.122", referrer: "https://192.168.50.122/"
